Jenkinsfile

```diff
@@ -632,7 +636,7 @@ pipeline{
 		count++;
 		    props = readProperties file:'data-hub/pipeline.properties';
 		    def reviewState=getReviewState()
-			if((env.CHANGE_TITLE.split(':')[1].contains("Automated PR")) || reviewState.equalsIgnoreCase("APPROVED")){
```

Comment:
We need to continue to merge state in case the PR is approved.

Reply:
there is `else if` condition down below. That would cover case if PR is approved?

`else if(reviewState.equalsIgnoreCase("APPROVED")){
			  sh 'exit 0'
}`